src/databricks/labs/ucx/source_code/python/python_ast.py

```diff
@@ -77,9 +76,7 @@ def maybe_parse(cls, code: str) -> MaybeTree:
             root = parse(code)
             tree = Tree(root)
             return MaybeTree(tree, None)
-        except AstroidSyntaxError as e:
-            return cls._definitely_failure('syntax-error', code, e)
-        except SystemError as e:
+        except Exception as e:  # pylint: disable=broad-exception-caught
```

Comment:
@nfx : Would have like to see a comment that motivates why we have a broad exception